### PR TITLE
fix(server,cli): don't float promises

### PR DESCRIPTION
**Pull request by etnoy**
Should this be done on the web, too?

### CODE REVIEW

File: server/src/domain/asset/asset.service.ts [hunk 1]
```diff
@@ -272,7 +272,7 @@ export class AssetService {
       zip.addFile(originalPath, filename);
     }
 
-    zip.finalize();
+    await zip.finalize();
```
Comment (jrasm91) [inline server/src/domain/asset/asset.service.ts:275]: This one breaks downloading files I believe. It needs to not be awaited, so I think we need to add a catch

**Review by jrasm91 — APPROVED** [review]
Minus the archive one this looks good, thank you.

File: server/src/domain/asset/asset.service.ts (hunk 1 above)
Comment (jrasm91) [reply server/src/domain/asset/asset.service.ts]: https://github.com/archiverjs/node-archiver/issues/476

**Review by jrasm91 — APPROVED** [review]

**Comment by etnoy** [comment]
> Minus the archive one this looks good, thank you.

I can't even remember if I fixed this one, could you check for me?

**Pull request merged**